beets/util/__init__.py

```diff
@@ -855,7 +855,9 @@ def open_anything():
     if sys_name == 'Darwin':
         base_cmd = 'open'
     elif sys_name == 'Windows':
-        base_cmd = 'start'
+        # python is not aware of cmd commands
+        # Start cmd.exe with start as argument
+        base_cmd = 'cmd /c \"start {}\"'
```

Comment:
Here's the last thing we need a way around: using interpolation will cause unpredictable problems here when the filename contains special characters (including spaces). Do you know whether it's possible to get the same effect while using proper argument lists?

For example, what I would *want* to work would be:

```py
subprocess.call('start', filename)
```

but, as you realized, that won't work because `start` is a cmd shell built-in. See [this SO answer](https://stackoverflow.com/questions/9137303/subprocess-popenstart-fails), for example.

Is there any clever way we can pass the filename argument as a proper shell argument without shell interpolation? As a last resort, we may need to escape the filename ourselves. :confused:

Reply:
``subprocess.call(["cmd", "/c", "start", file])``
I got this working